crates/ol/state-types/src/epochal.rs

```diff
@@ -55,7 +60,11 @@ impl EpochalState {
     /// Appends a new ASM manifest to the accumulator, also updating the last L1
     /// block height and other fields.
     pub fn append_manifest(&mut self, height: L1Height, mf: AsmManifest) {
-        // TODO actually insert into mmr
+        let manifest_hash = mf.blkid().as_ref();
+        self.manifest_mmr
+            .inner_mut()
+            .add_leaf(*manifest_hash)
```

Comment:
This is just appending the blkid from the manifest, it should be appending the manifest's hash tree root so that we record the logs that were emitted and we don't have to make an annoying proof to get the wtxs root.